relayer/testdata/curl.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
We should probably name this script to reflect what it does. It looks like it fetches test data so maybe `fetchData.sh`?